src/Standards/Generic/Tests/NamingConventions/ConstructorNameUnitTest.3.inc

```diff
@@ -0,0 +1,7 @@
+<?php
+
+// Intentional parse error (missing function name).
+// This should be the only test in this file.
+// Testing that the sniff is *not* triggered.
+
+function
```

Comment:
Could you please clarify why this test is needed? It is not immediately clear to me as it does not exercise the part of the sniff code that is changed in this PR.

Reply:
@rodrigoprimo Good question and I can't remember now and can see your point as this sniff code will only be triggered when a function is inside an OO structure.

Might just be that I was working on multiple related sniffs/fixer conflicts at the same time and missed that this one didn't need this test. Will remove.